ExportNeosToJson/ExportNeosToJson.cs

```diff
@@ -1,4 +1,4 @@
-﻿using BaseX;
+using BaseX;

```

Comment:
The reason the diff is doing weird stuff is because you're changing all the line endings you touch from LF to CRLF. Make sure your text editor is set to save using LF.

Reply:
i think i fixed it. 
running git ls-files --eol says its lf now.
`i/lf    w/lf    attr/text=auto          ExportNeosToJson/ExportNeosToJson.cs`
`i/lf    w/lf    attr/text=auto          ExportNeosToJson/Properties/AssemblyInfo.cs`
`i/lf    w/lf    attr/text=auto          README.md`